Dockerfile.standalone

```diff
@@ -0,0 +1,31 @@
+FROM node:12 AS build
+
+# This dockerfile does not require the app to be built on the host machine.
+# simply builds backstage in node image then transfers compiled files to nginx container.
+
+# You can build this image with `docker build -f Dockerfile.standalone -t backstage .`
+
+# Build backstage with node
+RUN npm install typescript -g
```

Comment:
this should really not be necessary, right? we should be using the typescript version that is in the project's dependencies

Reply:
Yes, You're right. Removing...